lib/c_glib/src/thrift/c_glib/transport/thrift_ssl_socket.h

```diff
@@ -214,5 +222,17 @@ thrift_ssl_socket_initialize_openssl(void);
 void
 thrift_ssl_socket_finalize_openssl(void);
 
+/**
+ * @brief authorize function
+ *
+ * It will do the authorization of ssl socket.
+ *
+ * It should be called in process of ssl socket handle handshake.
+ *
+ *
+ */
+void
+thrift_ssl_socket_authorize(void);
```

Comment:
This doesn't match the implementation: https://github.com/apache/thrift/blob/9f0548e05f1b1383cc122148d60cc40383ce599e/lib/c_glib/src/thrift/c_glib/transport/thrift_ssl_socket.c#L484